<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALM Assessments</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QLabel" name="reportLabel">
          <property name="text">
           <string>Select Report:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reportBox">
          <item>
           <property name="text">
            <string>Discount Rate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Neutral Risk</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="figuresLabel">
          <property name="text">
           <string>Available Figures:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="mplfigs"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QTableWidget" name="pandasWidget"/>
        </item>
        <item>
         <widget class="QWidget" name="plotWidget">
          <layout class="QVBoxLayout" name="mplvl">
           <item>
            <layout class="QVBoxLayout" name="mplvLayout"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <menubar class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <addaction name="actionParameters"/>
   </widget>
   <addaction name="menuParameters"/>
  </menubar>
  <statusbar class="QStatusBar" name="statusbar"/>
  <action name="actionParameters">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 